src/components/Input.vue

```diff
@@ -11,6 +12,7 @@
 export default {
   name: 'Input',
   props: {
+    dataTest: String,
```

Comment:
IMHO we should avoid add logic for tests in the component whenever we can. For instance, can we use input value in data-test attribute instead of pass an extra `dataTest` prop?